rust/chains/abacus-ethereum/src/inbox.rs

```diff
@@ -42,7 +42,9 @@ where
 {
     contract: Arc<EthereumInboxInternal<M>>,
     provider: Arc<M>,
+    #[allow(unused)]
```

Comment:
I assume you know the imapct of removing these because I don't :)

Reply:
Had to add the allow statements because the newer version has slightly better checks for unused values. We could in theory just remove the fields since they are, unused, but it felt wrong to do so when my only goal was to update the rust version.